tools/shared/src/main/scala/org/scalajs/core/tools/classpath/PartialClasspath.scala

```diff
@@ -151,6 +164,9 @@ object PartialClasspath {
   type DependencyFilter =
     Traversable[FlatJSDependency] => Traversable[FlatJSDependency]
 
+  type ManifestFilter =
+    Traversable[JSDependencyManifest] => Traversable[JSDependencyManifest]
```

Comment:
Yes, I know. I didn't know where to put this... It should be here by symmetry to `DependencyFilter`, but I need it in `ManifestFilters` which arguably should not know about the `classpath` package at all...